notebooks/text_models/solutions/keras_for_text_classification.ipynb

```diff
@@ -176,7 +176,7 @@
    "cell_type": "markdown",
```

Comment:
<p>In general, I agree but with the introduction of preprocessing layers, it's actually better to split the data first and then preprocess since the preprocessing layer directly feeds into the model. I followed the same order that is recommended on the TF tutorial page here (https://www.tensorflow.org/tutorials/keras/text_classification#prepare_the_dataset_for_training)</p>